File: api/v2010/outgoing_caller_id/update-default/update-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

outgoing_caller_id = @client
  .outgoing_caller_ids('PNaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .update(friendly_name: 'friendly_name')

puts outgoing_caller_id.sid

File: api/v2010/credential/update-default/update-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Sip.CredentialList;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var credential = CredentialResource.Update(
            pathCredentialListSid: "CLaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathSid: "CRaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(credential.Sid);
    }
}